Resources/Textures/Structures/Wallmounts/flags.rsi/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/d0d81185f09ca30d3b0856d476544240dba0de53, drawn by Saveliy360",
```

Comment:
doesn't seem too clear: what exactly was drawn by you and what was taken from /tg/